src/components/TransactionForm.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import FormButtons from './FormButtons';
+import 'antd/dist/antd.css';
+import { Input, message } from 'antd';
+import { ShopOutlined, TableOutlined, DollarOutlined } from '@ant-design/icons';
+
+class TransactionForm extends Component {
+    constructor() {
+        super()
+        this.state = {
+            amount: '',
+            vendor: '',
+            category: ''
+        }
+    }
+
+    handleInputChange = (e) => {
+        const target = e.target;
+        const value = target.value;
+        const name = target.name;
+
```

Comment:
use destructuring https://hacks.mozilla.org/2015/05/es6-in-depth-destructuring/

Reply:
will do